src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -44,4 +44,10 @@ public String getMockEndpoint() {
     public Event createEvent(@RequestBody Event newEvent) {
         return eventService.saveEvent(newEvent);
     }
+
+    // full path: /api/v1/events/{id}
+    @PutMapping("{id}")
+    public Event replaceEvent(@RequestBody Event newEvent, @PathVariable Long id) {
```

Comment:
Proper naming and parameters conventions; great job ✅

Reply:
I wasn't previously familiar with `PUT` endpoints, since I've mostly just worked with GET & POST, but after seeing this PR, I dug online to make sure you're following good practices.